src/plugins/kibana_overview/public/components/overview/overview.tsx

```diff
@@ -54,12 +54,19 @@ interface Props {
 export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) => {
   const [isNewKibanaInstance, setNewKibanaInstance] = useState(false);
   const {
-    services: { http, docLinks, data, uiSettings, application },
+    services: { http, docLinks, data, share, uiSettings, application },
   } = useKibana<CoreStart & AppPluginStartDependencies>();
   const addBasePath = http.basePath.prepend;
   const indexPatternService = data.indexPatterns;
   const IS_DARK_THEME = uiSettings.get('theme:darkMode');
 
+  // Home does not have a locator implemented, so hard-code it here.
+  const addDataHref = addBasePath('/app/home#/tutorial_directory');
```

Comment:
A side question: Is the lack of implementing a locator for `Home` something we should consider addressing? To me, it's not ideal to have to hard code the path.

Reply:
I agree that's it's something that should be implemented. #87304 was the original meta issue around locators, but I don't think there are issues for each plugin that should be implementing them. I didn't do it here because it's not my area of expertise and I wanted to keep the scope constrained.